docker-entrypoint.sh

```diff
@@ -86,4 +89,29 @@ if [ "$DISABLE_PGSQL" != "YES" ] && [ ! -f /run/postgresql/.init ]; then
   touch /run/postgresql/.init
 fi
 
+# init MongoDB
+echo "use admin
+db.createUser(
+  {
+    user: \"$MONGODB_USER\",
+    pwd: \"$MONGODB_PASSWORD\",
+         roles: [
+                   { role: \"userAdminAnyDatabase\", db: \"admin\" },
+                   { role: \"readWriteAnyDatabase\", db: \"admin\" },
+                   { role: \"dbAdminAnyDatabase\", db: \"admin\" },
+                   { role: \"clusterAdmin\", db: \"admin\" }
+                ]
+          }
+)" > /data/admin.js;
+mongod --dbpath /data/db run &
```

Comment:
should be moved to supervisor (pls check how other binaries are run in this repo)